cmd/operator-sdk/bundle/create.go

```diff
@@ -230,11 +232,21 @@ func (c bundleCreateCmd) runGenerate() error {
 		c.overwrite = true
 	}
 
+	annotationsFile := getAnnotationsFilePath(c.outputDir, c.directory)
```

Comment:
If there is a pre-existing `annotations.yaml`, the `bundle.ValidateAnnotation` inside `GenerateFunc` unmarshals the file and checks the length of entries in the map which is supposed to be 6. In order to pass the validation, we would have to remove any external entries.

Reply:
These labels need to be added to the `bundle.Dockerfile` as well. There shouldn't be a mismatch between annotations and labels, even though we aren't using docker labels, right @kevinrizza @dinhxuanvu?

Reply:
Correct. They should match.